source/base/parameter_handler.cc

```diff
@@ -1346,6 +1346,17 @@ bool ParameterHandler::read_input (std::istream &input,
   while (std::getline (input, input_line))
     {
       ++current_line_n;
+      // to make dealing with continuation lines easier, trim the whitespace
+      // at the beginning of every line.
+      std::string::size_type non_whitespace_index = 0;
+      for (; non_whitespace_index < input_line.size(); ++non_whitespace_index)
+        {
+          if (!std::isspace(input_line[non_whitespace_index]))
+            {
+              break;
+            }
+        }
+      input_line.erase(0, non_whitespace_index);
```

Comment:
because the current version saves `0.001` seconds by only calling erase once, of course. I'll clean this up.